src/api/operations.ts

```diff
@@ -219,14 +219,16 @@ export const drawEllipsePath = (config: {
   y: number | PDFNumber;
   xScale: number | PDFNumber;
   yScale: number | PDFNumber;
-}): PDFOperator[] => {
-  let x = asNumber(config.x);
-  let y = asNumber(config.y);
+  rotate: Rotation;
```

Comment:
This must be marked as optional or else it will be a breaking change.
```suggestion
  rotate?: Rotation;
```